magit-todos.el

```diff
@@ -327,6 +365,9 @@ is killed."
         (insert "\n"))
       (insert "\n"))))
 
+(defun magit-todos--insert-items ()
+  (mapcar #'magit-todos--insert-items-by-type '("TODO" "FIXME")))
```

Comment:
That's a cool idea. Note that it does mean rescanning all the files for each type, which you may not care about since you're using rg. But you could do a single scan with a combined regexp, then group them separately before insertion (-group-by and seq-group-by are very handy for this sort of thing).